Source/WebKit/UIProcess/Launcher/haiku/ProcessLauncherHaiku.cpp

```diff
@@ -26,12 +26,83 @@
 #include "config.h"
 #include "ProcessLauncher.h"
 
+#define __STDC_FORMAT_MACROS
```

Comment:
Is this still needed?

Reply:
Yes to get uint64 into a string(as a format specifier) we need this